test/functional/apps/dashboard_elements/controls/common/range_slider.ts

```diff
@@ -96,7 +96,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           width: 'medium',
         });
         expect(await dashboardControls.getControlsCount()).to.be(2);
-        const secondId = (await dashboardControls.getAllControlIds())[1];
+        const [firstId, secondId] = await dashboardControls.getAllControlIds();
+        await dashboardControls.clearControlSelections(firstId);
+        await dashboardControls.rangeSliderWaitForLoading(firstId);
```

Comment:
Since we no longer ignore invalid ranges, we need to clear the selections on the first control for the placeholders to appear in the second control.